coinstacks/ethereum/api/src/index.ts

```diff
@@ -24,7 +24,15 @@ app.get('/health', async (_, res) => {
   })
 })
 
-app.use('/docs', swaggerUi.serve, async (_req: Request, res: Response) => {
+import swaggerDocument from './swagger.json'
+const options = {
+  customCss: '.swagger-ui .topbar { display: none }',
+  customSiteTitle: 'ShapeShift API Docs',
+  customfavIcon: './favi-blue.png',
+}
+
+app.use('/coinstacks/common/api/public/favi-blue.png', express.static('./favi-blue.png'))
```

Comment:
I am not seeing the favicon after pulling down locally. Can you verify this again on your setup?

Reply:
I have it running on http://eu.scprimestats.com:31300/docs/

Reply:
similarly, i'm not seeing the icon running locally, but am seeing the page title change

Reply:
This is working for me locally
```const options = {
  customCss: '.swagger-ui .topbar { display: none }',
  customSiteTitle: 'ShapeShift API Docs',
  customfavIcon: '/favi-blue.png',
}

app.use(express.static(join(__dirname, '../../../../../../common/api/public')))```

Reply:
I do see the icon at your link, but looking at the code doesn't make sense to me. Saying it is serving the icon from `http://eu.scprimestats.com:31300/coinstacks/common/api/public/favi-blue.png` but there is no image there. Could there have been local changes making this work that aren't happening from the code (copy of the image in the swagger-ui-dist)?

Reply:
If you see image below on changes there's nothing. Only thing underlying is nginx is running on that server. 

![image](https://user-images.githubusercontent.com/35115310/133329988-028e9e0b-c898-4369-ab06-16fb0043c1ed.png)